test-cases/gutenberg/drag-and-drop-blocks.md

```diff
@@ -0,0 +1,375 @@
+<div class="wp-block-media-text alignwide is-stacked-on-mobile"><figure class="wp-block-media-text__media"></figure><div class="wp-block-media-text__content"><!-- wp:paragraph {"className":"has-large-font-size"} -->
+<p class="has-large-font-size"></p>
+<!-- /wp:paragraph --></div></div>
+<!-- /wp:media-text -->
+
+<!-- wp:heading {"level":4} -->
+<h4>Cover</h4>
+<!-- /wp:heading -->
+
+<!-- wp:cover {"url":"https://cldup.com/cXyG__fTLN.jpg","id":890,"dimRatio":20,"overlayColor":"luminous-vivid-orange","focalPoint":{"x":"0.63","y":"0.83"},"minHeight":219,"isDark":false} -->
+<div class="wp-block-cover is-light" style="min-height:219px"><span aria-hidden="true" class="wp-block-cover__background has-luminous-vivid-orange-background-color has-background-dim-20 has-background-dim"></span><img class="wp-block-cover__image-background wp-image-890" src="https://cldup.com/cXyG__fTLN.jpg" style="object-position:63% 83%" data-object-fit="cover" data-object-position="63% 83%"/><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"align":"center","placeholder":"Write title…","className":"has-text-color has-very-light-gray-color","fontSize":"large"} -->
+<p class="has-text-align-center has-text-color has-very-light-gray-color has-large-font-size">Cool cover</p>
+<!-- /wp:paragraph --></div></div>
+<!-- /wp:cover -->
+
+<!-- wp:heading -->
+<h2>Dynamic Blocks</h2>
+<!-- /wp:heading -->
+
+<!-- wp:latest-posts {"displayPostContent":true,"displayPostDate":true} /-->
+
+<!-- wp:heading -->
+<h2>Buttons</h2>
+<!-- /wp:heading -->
+
+<!-- wp:buttons -->
+<div class="wp-block-buttons"><!-- wp:button -->
+<div class="wp-block-button"><a class="wp-block-button__link">Solid Button</a></div>
+<!-- /wp:button -->
+
+<!-- wp:button {"gradient":"luminous-vivid-amber-to-luminous-vivid-orange"} -->
+<div class="wp-block-button"><a class="wp-block-button__link has-luminous-vivid-amber-to-luminous-vivid-orange-gradient-background has-background">Gradient Button</a></div>
+<!-- /wp:button --></div>
+<!-- /wp:buttons -->
+
+<!-- wp:heading -->
+<h2>Legacy</h2>
+<!-- /wp:heading -->
+
+<!-- wp:shortcode -->
+[youtube https://www.youtube.com/watch?v=ssfHW5lwFZg]
+<!-- /wp:shortcode -->
+
+<!-- wp:heading -->
+<h2>Unsupported</h2>
+<!-- /wp:heading -->
+
+<!-- wp:rss /-->
+
+<!-- wp:heading {"textAlign":"left","level":4,"style":{"typography":{"lineHeight":"2.5"}},"className":"has-primary-background-color has-background"} -->
+<h4 class="has-text-align-left has-primary-background-color has-background" style="line-height:2.5">Heading with line-height set</h4>
+<!-- /wp:heading -->
+```
+
+</details>
+
+The following QR code can be used for accessing this page easily from a device:
+
+<img src="https://chart.googleapis.com/chart?cht=qr&chs=500x500&chl=https%3A%2F%2Fgithub.com%2Fwordpress-mobile%2Ftest-cases%2Fblob%2Ftrunk%2Ftest-cases%2Fgutenberg%2Fdrag-and-drop-blocks.md%23html-example" width="150" height="150">
```

Comment:
Note that the URL used points to `trunk` as it will be the final URL once this PR is merged.